addons/mail/static/src/components/message/message.scss

```diff
@@ -64,6 +64,8 @@
 
 .o_Message_core {
     min-width: 0; // allows this flex child to shrink more than its content
+    margin-top: map-get($spacers, 2);
+    margin-bottom: map-get($spacers, 2);
     margin-inline-end: map-get($spacers, 3);
```

Comment:
So I guess if you just increase this to 4 or 5 you could remove the padding-right from `o_Message`? To check which one looks best.

Reply:
@seb-odoo Done!!